### DISCUSSION TITLE
FYI - Special characters at beginning of passwords

### DISCUSSION THREAD

**Discussion by lowgatelabs** [philosowaffle/peloton-to-garmin#163] [General]
As an FYI... My peloton password had a special character at the beginning as I use a password manager. This apparently was causing p2g to fail silently, and all I'd get in the log was "Peloton To Garmin."

After banging my head against the wall, I finally saw that character and changed my PW to start with an alphanumeric character and all was well.

Once working, this is AWESOME and I look forward to no long having to remember to start/stop my garmin and only get half-baked metrics after a peloton ride!

**Comment by philosowaffle** [comment]
Because the config is written in JSON, some special characters may need to be [properly escaped](https://stackoverflow.com/a/27516892) before they can be parsed.  [Here is a tool](https://jsonlint.com/?code=) that you can use to validate your config is valid json.
